argus/backend/service/argus_service.py

```diff
@@ -234,7 +246,7 @@ def assign_runs_for_scheduled_group(self, schedule: ArgusSchedule, group_id: UUI
                 row.save()
 
     def submit_new_schedule(self, release: str | UUID, start_time: str, end_time: str, tests: list[str | UUID],
-                            groups: list[str | UUID], assignees: list[str | UUID], tag: str) -> dict:
+                            groups: list[str | UUID], assignees: list[str | UUID], tag: str, comments: dict[str, str] | None, group_ids: dict[str, str] | None) -> dict:
```

Comment:
what are `groups` and `group_ids`?

Reply:
This method is used by both group and test planner,  so for test planner, `group_ids` are just hints to make it possible to create comments there (could be folded into single parameter, e.g. `comments`), `groups` on the other hand is only used when using Weekly Planner and that is an array of groups to schedule (much like `tests` is an array of tests to schedule).